src/jit/function-builder.cc

```diff
@@ -391,6 +455,26 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
       break;
     }
 
+    case Opcode::Br: {
+      auto target = &thread_->GetIstream()[ReadU32(&pc)];
+      auto it = std::find_if(workItems_.cbegin(), workItems_.cend(), [&](BytecodeWorkItem const& b){
```

Comment:
For consistency, please add a space between `)` and `{` and put the `const` specifier before the type name (same problem in `InterpBrUnless`).